04-advanced-topics/validate_differential_expression_adv_topics_01.Rmd

```diff
@@ -391,10 +391,10 @@ test.gene <- refine.bio.df %>%
 box.plot <- data.frame(test.gene, subgroup = metadata$subgroup) %>%
```

Comment:
Same comment here re explicitly attaching the ggplot2 library at the top of this notebook.

Reply:
Same.